### PR TITLE
Fixes pep8 warnings for python modules

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes https://github.com/projectatomic/atomic/issues/115

### CODE REVIEW

File: Atomic/__init__.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import sys
-import os
```
Comment (navidshaikh) [inline Atomic/__init__.py]: removed unsued `os`

File: Atomic/atomic.py [hunk 2]
```diff
@@ -1,14 +1,12 @@
 import sys
 import os
 import argparse
-import gettext
```
Comment (navidshaikh) [inline Atomic/atomic.py]: removed unsued imports

File: Atomic/atomic.py [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
 import docker
 import json
 import subprocess
 import getpass
 import requests
 import pipes
-import selinux
```
Comment (navidshaikh) [inline Atomic/atomic.py]: removed unsued imports

File: Atomic/atomic.py [hunk 4]
```diff
@@ -310,7 +307,6 @@ def _get_args(self, label):
     def _check_latest(self):
         inspect = self._inspect_image()
         if inspect and inspect["Id"] != self.inspect["Image"]:
-            response = ""
```
Comment (navidshaikh) [inline Atomic/atomic.py]: F841 local variable 'response' is assigned to but never used

File: Atomic/atomic.py [hunk 5]
```diff
@@ -671,7 +667,6 @@ def get_label(label):
             version = "%s-%s-%s" % (current_name, get_label("Version"),
                                     get_label("Release"))
 
-        prev = ""
```
Comment (navidshaikh) [inline Atomic/atomic.py]: F841 local variable 'prev' is assigned to but never used

File: atomic_dbus.py [hunk 6]
```diff
@@ -3,10 +3,9 @@
 import dbus
 import dbus.service
 import dbus.mainloop.glib
-from gi.repository import GObject, GLib
```
Comment (navidshaikh) [inline atomic_dbus.py]: removed unsued import

File: Atomic/atomic.py [hunk 7]
```diff
@@ -671,7 +667,6 @@ def get_label(label):
             version = "%s-%s-%s" % (current_name, get_label("Version"),
                                     get_label("Release"))
```
Comment (navidshaikh) [inline Atomic/atomic.py]: variable `version` is assigned but never used, should there be a reason to keep it?

File: Atomic/__init__.py [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
 from .pulp import PulpServer
 from .config import PulpConfig
 from .atomic import Atomic
```
Comment (navidshaikh) [inline Atomic/__init__.py]: `PulpConfig` and `Atomic` are imported but unused, should these imports be removed?

**Comment by willmtemple** [comment]
@navidshaikh What's the linter that generated these changes? I had previously run `pep8` over some of this code and it produced different results.

@rhatdan LGTM

**Comment by navidshaikh** [comment]
@willmtemple : `pep8` and `flake8` linter checkers generated a set of warnings.

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
Can you send a patch to run these tests on make test.

**Comment by willmtemple** [comment]
@rhatdan We *could* look into a more comprehensive static analysis tool like landscape.io, which is what github.com/projectatomic/atomicapp uses, which would give us a more complete "health report" including code smells and style errors like these.

**Comment by rhatdan** [comment]
Sounds good to me.

**Pull request merged**